components/01-atoms/date-time/date-time.twig

```diff
@@ -11,4 +11,4 @@
 {# Expample: April 1, 2022 #}
 {% set date_time = date_time__start|date(format__date) %}
 
-{{ date_time }}
+<span {{ bem('date-time') }}>{{ date_time }}</span>
```

Comment:
I believe times/dates should be represented with `<time>` element. So in this case you would use `<time datetime="2022-01-25">January 25, 2022</time>` see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time